e2e-tests/security-context/run

```diff
@@ -10,6 +10,13 @@ create_namespace $namespace
 deploy_operator
 
 kubectl_bin apply -f $conf_dir/secrets.yml -f $conf_dir/client.yml -f $conf_dir/minio-secret.yml
+kubectl_bin apply -f "$test_dir/conf/service-account.yml"
+if [[ "${OPENSHIFT}" == 1 ]]; then
+    oc adm policy add-scc-to-user privileged -z percona-server-mongodb-operator
```

Comment:
why do we need to make a 'privileged' this user?

Reply:
Openshift won't let you change anything under securityContext field if you don't belong to role with some additional grants

Reply:
please check this PR https://github.com/percona/percona-xtradb-cluster-operator/commit/627123c32fdf0fbc331ae303c1316c96fa9c996b

Reply:
upgraded